src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age && (<p className="Person__age">{`I am ${person.age}`}</p>)}
```

Comment:
Too long string, split into parts. A common mistake in your code for this component.